src/migration.ts

```diff
@@ -78,6 +79,11 @@ export async function _migrateMigrationSchema(
   `);
 }
 
+export const isMigrationFilename = (
+  filename: string
+): RegExpMatchArray | null =>
+  /^[0-9]{6,}(-[-_0-9A-Za-z]+)?\.sql$/.exec(filename);
```

Comment:
`000001-.sql` is poorly named, but should it be considered a migration? I think it probably should?

```suggestion
  /^[0-9]{6,}(-[-_0-9A-Za-z]*)?\.sql$/.exec(filename);
```